components/form-builder/panel/ElementPanel.tsx

```diff
@@ -169,6 +179,27 @@ const Form = ({ item }: { item: ElementTypeWithIndex }) => {
   return (
     <>
       <Row>
+        <div>
+          <QuestionNumber>{item.index + 1}</QuestionNumber>
```

Comment:
Should this use `item.id` instead?

Reply:
I think the number should be the consistent index, not the "id" of the question. 
I agree that it's a bit disorienting, but the numbers should represent the order in which someone filling out the form will actually see it.